lib/words.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+function words() {
+
+	this.adj = function() {
+		this.confused = true;
+		this.curious = true;
+		this.fancy = true;
+		this.dull = true;
+		this.fluttering = true;
+	};
+
+	this.vrb = function() {
```

Comment:
these should each be in their own file. So, a file each for adjectives, nouns, verbs, and for each function.